src/podman.scss

```diff
@@ -24,6 +24,12 @@
           display: none;
         }
     }
+
+    // Labels should not stretch
```

Comment:
This comment is misleading. This affects not really the labels but the while rows. 
In addition it's a really intrusive css override which needs to be explained and if it's generic enough moved to our overrides file and upstreamed.

Reply:
This is also only interesting for mobile mode.

Reply:
> This affects not really the labels

It's literally talking about _data-**label**_:
    
```css
.pf-m-grid-md.pf-c-table [data-label] {
```

> This is also only interesting for mobile mode.

Yes; it does not affect desktop. I figured having a media query here where it doesn't affect desktop doesn't matter.

> it's a really intrusive css override 

It's inside of the following CSS, so it's not intrusive whatsoever (as it only apples to the containers and images lists):
```
#containers-images, #containers-containers {
```

> which needs to be explained

`justify-items` is part of CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items

And the start value is:

"The item is packed flush to each other toward the start edge of the alignment container in the appropriate axis."

Reply:
There isn't a `justify-items: start` equivalent on the PF4 utilities @ https://www.patternfly.org/v4/utilities/flex (which are called "flex" but also apply to grid too), so this one CSS line is necessary.

Reply:
data-label != label and since the data-label attr applies on the whole table rows, this comment is indeed misleading, maybe not for you that you wrote it of course. Please change it to 'Badges should not stretch on mobile mode' or something that we can all understand what it fixes when looking at the CSS.

In addition - I know that this override exists as is also in cockpit-machines and therefore I asked if it's generic enough to be upstreamed, or at least put into our overrides file.

Reply:
Aha, I get it. Will do. Thanks!

Reply:
I thought you meant it's generic and would be applied where we don't want it. I didn't read it right at first, but yeah, I think you're right that this might be better as a generic fix in Cockpit, as elsewhere will probably have this issue. And, yes, we should report it upstream to PF.

My _guess_ is that they probably have a reason for this, perhaps with widgets in mind, so buttons and dropdowns would take up the space.